apps/service-portal/src/components/ApplicationErrorBoundary/ApplicationErrorBoundary.tsx

```diff
@@ -45,13 +53,49 @@ const Error: FC = () => {
   const { formatMessage } = useLocale()
 
   return (
-    <Box>
-      <Text variant="h2" as="h2">
-        {formatMessage({
-          id: 'sp:error-page-heading',
-          defaultMessage: 'Eitthvað fór úrskeiðis',
-        })}
-      </Text>
+    <Box marginTop={8}>
+      <Box
+        marginTop={[0, 6]}
+        marginBottom={[0, 6]}
+        textAlign="center"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <Box marginBottom={4}>
+          <Tag variant="red">{500}</Tag>
+        </Box>
+        <Text variant="h1" as="h1" marginBottom={3}>
+          {formatMessage({
+            id: 'sp:error-page-heading',
+            defaultMessage: 'Eitthvað fór úrskeiðis',
+          })}
+        </Text>
+        <Text variant="default" as="p">
+          {formatMessage({
+            id: 'sp:error-page-text',
+            defaultMessage:
+              'Því miður hefur eitthvað farið úrskeiðis og ekki næst samband við vefþjón.',
+          })}
+        </Text>
+        <Box marginTop={2}>
+          <a href="https://island.is" target="_blank" rel="noreferrer">
+            <Button variant="text" size="medium">
+              Ísland.is
+            </Button>
+          </a>
+        </Box>
+      </Box>
+
+      <Box display="flex" justifyContent="center" marginBottom={[1, 0]}>
+        <img
+          src="./assets/images/hourglass.svg"
+          alt={formatMessage({
+            id: 'sp:error-page-img-alt',
+            defaultMessage: 'Skrautmynd fyrir villuskjá',
+          })}
```

Comment:
```suggestion
          alt=""
```

Use empty alt text for decorative images